### ISSUE TITLE
Need help for HTTP tunnel

### ISSUE THREAD

**Issue by andrewheadricke** [eyedeekay/goSam#6]
I've tried this and a bunch of other Go SAM libraries and I can't seem to get it working. I can connect to the local i2pd node and it provides me with a b32 endpoint, however when I try to use a browser to connect to that endpoint it never seems to work. The goSam bridge never notifies me of any connection.

Is this even possible? Can an I2P configured browser access SAM .b32.i2p endpoints?

**Comment by eyedeekay** [comment]
Yes, the example in the README is exactly how that is done for an HTTP Client. goSam implements a Dialer, which can be a part of a Transport, which can be replaced in an HTTP Client. To build something a browser can connect to, however, will require further elaboration. You can, for instance, replace the dialer and resolver in https://github.com/getlantern/go-socks5 with the dialer and resolver from goSam to create a TCP SOCKS proxy. If you need an HTTP Proxy, then adapting one of these: https://github.com/eyedeekay/httptunnel may be a better choice.
